src/components/alerting.tsx

```diff
@@ -1902,9 +1977,32 @@ const silenceStateOrder = (silence: Silence) => [
   _.get(silence, silenceState(silence) === SilenceStates.Pending ? 'startsAt' : 'endsAt'),
 ];
 
+const SelectAllCheckbox: React.FC<{ silences: Silence[] }> = ({ silences }) => {
+  const { selectedSilences, setSelectedSilences } = React.useContext(SelectedSilencesContext);
+
+  const activeSilences = _.filter(silences, (s) => silenceState(s) !== SilenceStates.Expired);
```

Comment:
should we avoid using lodash references from new code so when we refactor it away is easier?

Reply:
My understanding was that we're not aiming to completely refactor away our use of Lodash, but just to refactor away places where it is used with very little benefit. (Arguably this use of `_.filter` fits that definition 😅). We should definitely reduce our use of Lodash, but not sure where the line should be drawn.

Reply:
I think we could agree on prefer to use native functions when possible, then we could use lodash for something that provided by default. WDYT?

Reply:
Sounds good 👍 
There are some places where there really is no value added by using Lodash, so let's start be refactoring those away.